k8s/agent/deploy-agent.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tracetest-agent
+  labels:
+    app: tracetest-agent
+spec:
+  selector:
+    matchLabels:
+      app: tracetest-agent
+  template:
+    metadata:
+      labels:
+        app: tracetest-agent
+    spec:
+      containers:
+        - name: tracetest-agent
+          image: kubeshop/tracetest-agent:v0.13.9
```

Comment:
Thinking also about the version: is there a way to make this version configurable? My idea is to think a way to install a specific version of the agent in a cluster.

Reply:
Done

Reply:
By default, `kubeshop/tracetest-agent:latest` will be used. 

If the user provides a third option with the version, that version will be used instead:

```
curl <path> | bash -s -- <namespace> <api-key> (<version>?)
```